packages/reports/src/Containers/open-positions.tsx

```diff
@@ -480,7 +484,10 @@ const OpenPositions = ({
     const [accumulator_rate, setAccumulatorRate] = React.useState(accumulator_rates[0]);
     const is_accumulator_selected = contract_type_value === contract_types[2].text;
     const is_multiplier_selected = contract_type_value === contract_types[1].text;
-    const contract_types_list = contract_types.map(({ text }) => ({ text, value: text }));
+    const show_accu_in_dropdown = !is_eu && is_virtual;
+    const contract_types_list = contract_types
+        .filter(contract_type => contract_type.value !== 'accumulators' || show_accu_in_dropdown)
```

Comment:
```suggestion
        .filter(contract_type => contract_type.value !== localize('Accumulators') || show_accu_in_dropdown)
```
@henry-deriv nit: in this case we won't need to add a `value` for each object in `contract_types`. :)

Reply:
sure